NBitcoin/RPC/RPCClient.cs

```diff
@@ -1284,9 +1284,9 @@ public async Task<Block> GetBlockAsync(uint256 blockId)
 		/// </summary>
 		/// <param name="blockId"></param>
 		/// <returns></returns>
-		public Block GetBlock(uint256 blockId)
+		public Block GetBlock(uint256 blockId, int verbosity = 0)

```

Comment:
should this be an int-based enum instead?

Reply:
https://github.com/MetacoSA/NBitcoin/pull/787#discussion_r366527035

Reply:
1. `IBlockRepository` will probably go away, this has not been very useful interface for this library
2. Adding optional parameter is a breaking change which requires rebuilding libraries depending on the version not using the parameter.